text/0670-aws-applicationsignals-enablement-l2.md

```diff
@@ -0,0 +1,633 @@
+# AWS CloudWatch Application Signals L2 Construct for Simplifying Enablement
+
+* **Original Author(s):**: @bjrara
+* **Tracking Issue**: [#670](https://github.com/aws/aws-cdk-rfcs/issues/670)
+* **API Bar Raiser**: @moelasmar
+
+The `application-signals-integration` module is a collection of L2 constructs which leverages native L1 CFN resources, simplifying the enablement steps
+and the creation of Application Signals resources. This addresses key challenges in the current CDK enablement process, which requires complex manual
+configurations for ECS customers, including container setup, volume management, and extensive environment variable configuration. Application Signals
+is designed to be flexible and is supported for other platforms as well. However, the initial focus is on supporting ECS, with plans to potentially
+extend support to other platforms in the future.
+
+## Working Backwards
+
+### CHANGELOG
+
+`feat(aws-applicationsignals-alpha): introduce Application Signals L2 constructs for simplifying the enablement steps`
+
+### README
+
+#### ApplicationSignalsTaskDefinition
+
+`ApplicationSignalsTaskDefinition` is a construct to update an existing TaskDefinition with Application Signals auto-instrumentation configurations,
+and attach required IAM permission **CloudWatchAgentServerPolicy** to the Task role.
```

Comment:
could you please add a link to this policy in AWS Docs

Reply:
Added.